styles/web/sass/core/widgetscustom/_progress-bar-theme.scss

```diff
@@ -5,39 +5,94 @@
 // To customize any core styling, copy the part you want to customize to styles/web/sass/app/ so the core styling is overwritten.
 //
 
-/* Based on https://github.com/mendixlabs/star-rating/blob/v1.1.1/src/ui/StarRating.scss */
-div.widget-progress-bar .progress-bar-default {
+/* ==========================================================================
+   Progress Bar
+
+   Default Mendix Progress Bar Widget.
+========================================================================== */
+
+.progress {
```

Comment:
Seems that this was brought into the widget's styling file, as opposed to being in bootstrap.